ibctest/docker.go

```diff
@@ -62,10 +63,10 @@ func handleDockerBuildOutput(t *testing.T, body io.Reader) {
 
 		_ = json.Unmarshal([]byte(line), &logLine)
 		if logLine.Stream != "" {
-			t.Log(logLine.Stream)
+			fmt.Print(logLine.Stream)
 		}
 		if logLine.Aux != nil {
-			t.Log(logLine.Aux)
+			fmt.Print(logLine.Aux)
```

Comment:
Functions associated with a particular test should log to that test rather than print to stdout. Was this change intentionally committed?

Reply:
Ah yes, meant to make a separate PR for this. the logLine.Stream and Aux is formatted for stdout. Using `fmt.Print` makes it print the same way a `docker build` would. For comparison:

`t.Log`:
![Screenshot from 2022-07-12 19-58-30](https://user-images.githubusercontent.com/6722152/178795167-8fca6298-defe-4906-a8e6-a136ff603771.png)

`fmt.Print`:
![Screenshot from 2022-07-12 19-59-21](https://user-images.githubusercontent.com/6722152/178795197-93266955-c9f0-4142-a897-7a8aba9fd7c7.png)